packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel_cloud_firestore.dart

```diff
@@ -0,0 +1,356 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:meta/meta.dart' show visibleForTesting, required;
+
+import '../cloud_firestore_platform_interface.dart';
+
+class MethodChannelCloudFirestore extends CloudFirestorePlatform {
+  MethodChannelCloudFirestore() {
+    channel.setMethodCallHandler(_handlePlatformCall);
+  }
+
+  @visibleForTesting
+  static const MethodChannel channel = MethodChannel(
+    'plugins.flutter.io/cloud_firestore',
+  );
+
+  // Platform calls
+  Future<void> _handlePlatformCall(MethodCall call) async {
+    switch(call.method) {
+      case 'QuerySnapshot':
+        return _handleQuerySnapshot(call);
+        break;
+      // case 'DocumentSnapshot':
```

Comment:
Reminder to handle this

Reply:
Yes, added in the latest revision of this PR